curriculum/challenges/english/25-front-end-development/lab-lunch-picker-program/66db529d37ad966480ebb633.md

```diff
@@ -353,6 +334,39 @@ try {
 
 ```
 
+This test ensures that the `getRandomLunch` function does not mutate the original input array, regardless of the selected random index.
```

Comment:
you should write an hint that is useful to the camper to know why this test is failing, like "When `getRandomLunch` is called, it should not mutate the array passed as argumetn" or something

this would be incredibly confusing to get as an hint